### PR TITLE
Enable volume resize e2e

### PR DESCRIPTION
**Pull request by meghnasiingh**
Signed-off-by: Meghna Singh <Meghna.Singh1@ibm.com>

### CODE REVIEW

File: tests/e2e/e2e.sh [hunk 1]
```diff
@@ -138,6 +138,11 @@ ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[sc\]" ./tests/e2e | tee -a block-vpc-c
 rc1=$?
 echo "Exit status for basic volume test: $rc1"
 
+ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[resize\] \[pv\]" | tee -a block-vpc-csi-volume-resize-ginkgo-log.txt
+rc3=$?
+echo "Exit status for resize volume test: $rc3"
+
+
```
Comment (ambiknai) [inline tests/e2e/e2e.sh:145]: Meghna, we need to check rc3 value below and report status accordingly.  [ we have some if condition already for rc1 && rc2 ]

File: tests/e2e/conf/testcases.json [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+    "VPC-BLK-CSI-TEST: Custom SC Pod Test",
+    "VPC-BLK-CSI-TEST: 5iops SC POD Test and resize volume"
+  ],
+  "drain": [
+    "VPC-BLK-CSI-TEST: Cordon and Drain Node"
+  ],
+  "statefulset": [
+    "VPC-BLK-CSI-TEST: With Statefulsets"
+  ]
+}
```
Comment (ambiknai) [inline tests/e2e/conf/testcases.json:20]: Could you please remove this new line character

**Review by ambiknai — APPROVED** [review]

**Review by arahamad — APPROVED** [review]

File: tests/e2e/e2e.sh [hunk 3]
```diff
@@ -138,6 +138,11 @@ ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[sc\]" ./tests/e2e | tee -a block-vpc-c
 rc1=$?
 echo "Exit status for basic volume test: $rc1"
 
+ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[resize\] \[pv\]" | tee -a block-vpc-csi-volume-resize-ginkgo-log.txt
```
Comment (sameshai) [inline tests/e2e/e2e.sh:141]: ginkgo -v -nodes=1 --focus="\[ics-e2e\] \[resize\] \[pv\]"  ./tests/e2e  | tee -a block-vpc-csi-volume-resize-ginkgo-log.txt

**Review by sameshai — CHANGES_REQUESTED** [review]
add ./tests/e2e

**Review by sameshai — APPROVED** [review]

**Pull request merged**